packages/core-graphql/package.json

```diff
@@ -29,6 +29,7 @@
     "dependencies": {
         "@arkecosystem/core-container": "^2.1.0",
         "@arkecosystem/core-http-utils": "^2.1.0",
+        "@arkecosystem/core-transaction-pool": "^2.1.0",
```

Comment:
Packages itself should never have a dependency on anything but interfaces and the container.